go.mod

```diff
@@ -18,3 +18,5 @@ require (
 	gorm.io/driver/postgres v1.1.0
 	gorm.io/gorm v1.21.9
 )
+
+replace github.com/flyteorg/flyteidl => github.com/hamersaw/flyteidl v0.19.26-0.20211103115633-100abab11c51
```

Comment:
Reminder to revert this when idl is merged

Reply:
done